WiserTaskScheduler/WiserTaskScheduler/Modules/Branches/Services/BranchQueueService.cs

```diff
@@ -2054,12 +2066,15 @@ SELECT id
                             }
                             case "REMOVE_LINK":
                             {
-                                // Check if the user requested this change to be synchronised.
-                                if (!entityTypeMergeSettings.Update)
+                                // check if the link type is in the list of changes 
```

Comment:
Please start the comment with a capital letter and end with a period.

Reply:
Done.